eventmesh/Etcd3EventMeshServer.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const config = require('../broker/lib/config');
+const logger = require('../broker/lib/logger');
+const CONST = require('../broker/lib/constants');
```

Comment:
Import logger and CONST directly from common.

Reply:
done